src/main/java/com/microservicio/covid/controller/NewsController.java

```diff
@@ -19,67 +16,33 @@
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
-import java.text.ParseException;
-import java.text.SimpleDateFormat;
-import java.util.Locale;
-
 @RestController
 @RequestMapping("/v1")
 public class NewsController {
 
-    public Logger LOGGER = LoggerFactory.getLogger(NewsController.class);
-    private String DATE_FORMAT = "yyyy-MM-dd";
+    private NewsWrapper newsWrapper = new NewsWrapper();
     @Autowired
     private NewsService service;
 
-    /*
-
-     */
     @GetMapping(value = "/news", produces = MediaType.APPLICATION_JSON_VALUE)
     @ApiOperation(value = "Retrieve News information from provider.",
             notes = "News form Argentinian country.", response = News.class)
     @ApiResponses(value = {@ApiResponse(code = 200, message = "OK")})
-    public ResponseEntity getNews(
-            @ApiParam(name = "published", value = "Date published of news.", required = true) @RequestParam(required = true) String published) throws Exception {
-
-        NewsWrapper newsInformation;
-
-        try {
-            NewsDTO newsDTO = new NewsDTO();
-            SimpleDateFormat format = new SimpleDateFormat(DATE_FORMAT, Locale.getDefault());
-            newsDTO.setPublished(format.parse(String.valueOf(published)));
-            newsInformation = service.getNewsWrapperData(newsDTO);
-            return new ResponseEntity<>(newsInformation, HttpStatus.OK);
-        } catch (ParseException  e) {
-            LOGGER.error("Error parsing {} variable.", published);
-            return new ResponseEntity(e, HttpStatus.INTERNAL_SERVER_ERROR);
-
-        } catch (NullPointerException e) {
-            LOGGER.error("The variable {} can not be null", published);
-            return new ResponseEntity(e, HttpStatus.INTERNAL_SERVER_ERROR);
-        } catch ( IllegalArgumentException e) {
-            LOGGER.error("The variable {} can not be null", published);
-            return new ResponseEntity(e, HttpStatus.INTERNAL_SERVER_ERROR);
-        }
+    public ResponseEntity<NewsWrapper> getNews(
+            @ApiParam(name = "published", value = "Date published of news.", required = true) @RequestParam(required = true) String published){
+            newsWrapper = service.getNewsWrapperData(published);
+            return new ResponseEntity<>(newsWrapper, HttpStatus.OK);
```

Comment:
```suggestion
            return new ResponseEntity<>(service.getNewsWrapperData(published), HttpStatus.OK);
```
newsWrapper podria ser eliminada, no recomiendo igualmente usar la misma variable compartida en todos los métodos

Reply:
Done.